platform/stub-setup.ts

```diff
@@ -1,13 +1,15 @@
 /*
- * Injects the stubs into the global scope.
- * Bundled HomeApp cannot be loaded until these globals are set.
+ * Injects the App stub into the global scope.
+ * Bundled HomeApp cannot be loaded until this global is set.
  */
-import { DeviceManagerStub } from './device-manager';
 import { AppStub } from './smart-home-app';
 
 (global as any).smarthome = {
   App: AppStub,
-  DeviceManager: DeviceManagerStub,
+  Intents: {
```

Comment:
can we type this with https://github.com/actions-on-google/local-home-sdk/blob/9840f62601fdefcf9144de4f0115e9a71c70e07f/execution-only.d.ts#L45 ?

Reply:
Yeah, I was wondering about this.  Since I'm setting the `smarthome` namespace I can't access `smarthome.Intents` from the SDK typings.  I guess I could declare it `Intents: smarthome.Intents`

Reply:
Guess I can't do this since it's a type.  Seems redundant that I need to redefine Intents here.

Reply:
Something like this:
```
const smarthomeStub: {
  App: smarthome.App
  DeviceManager: smarthome.DeviceManager
  Intents: {[key in keyof typeof smarthome.Intents]: string}
} = {
  App: AppStub,
  DeviceManager: DeviceManagerStub,
  Intents: {
    IDENTIFY: 'action.devices.IDENTIFY',
    EXECUTE: 'action.devices.EXECUTE',
  }
};
```
Could introduce a little bit more typing and help you catch if you're forgetting any enum keys. (This wouldn't let you typecheck against enums values since those seems to be opaque by design https://github.com/microsoft/TypeScript/issues/17690#issuecomment-321319291)

This might be worth raising an issue so that the SDK typing expose this using a proper union type instead.

Reply:
Thanks, I added the typing.  Can you elaborate on the use of a "proper union type"? Are you suggesting the SDK expose `smarthome.Intents` as a union type of string literals?